pytest_base_url/plugin.py

```diff
@@ -22,6 +19,12 @@ def base_url(request):
 @pytest.fixture(scope="session", autouse=True)
 def _verify_url(request, base_url):
     """Verifies the base URL"""
+
+    # Lazy load requests to reduce cost for tests that don't use the plugin
+    import requests
```

Comment:
Wrong place, should be within the condition

Reply:
Done. @RonnyPfannschmidt